apps/machine/zynq7/platform_info_remoteproc_master.c

```diff
@@ -31,7 +31,7 @@ extern struct hil_platform_ops zynq_a9_proc_ops;
 #define SHM_SIZE                          0x00200000
 #define VRING0_IPI_VECT                   6
 #define VRING1_IPI_VECT                   3
-#define MASTER_CPU_ID                     0
+#define MAIN_CPU_ID                       0
```

Comment:
`HOST` seems to be used more often when paired with `REMOTE`, such as in `RPMSG_HOST`.

Is this not used anywhere? If not, better to just drop these two defines.

Reply:
the `REMOTE_CPU_ID` is used, so I have kept it for readability

Reply:
Okay, then still would suggest `HOST_CPU_ID` if these defines need to remain.